CONTRIBUTING.md

```diff
@@ -49,6 +49,12 @@ See
 [`tox.ini`](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/master/tox.ini)
 for more detail on available tox commands.
 
+### Benchmarks
+
+Performance progression of benchmarks for packages distributed by OpenTelemetry Python can be viewed as a [graph of throughput vs commit history](https://open-telemetry.github.io/opentelemetry-python-contrib/benchmarks/index.html). From this page, you can download a JSON file with the performance results.
```

Comment:
```suggestion
Performance benchmarks for packages distributed by OpenTelemetry Python can be viewed as a [graph of throughput vs commit history](https://open-telemetry.github.io/opentelemetry-python-contrib/benchmarks/index.html). From this page, you can download a JSON file with the performance results.
```

Throughput I'm a bit confused about. What does that mean? number of iterations over some time period?

Reply:
Suggesting against "progression" as I think that implies that we are trying to trend benchmarks one direction or another. I don't think we're consciously trying to improve performance, but we are trying to monitor it so it doesn't get out of control.

Reply:
From what I understand, `pytest-benchmark` runs the test as many times as it can in a second (this is configurable). So the graph this action outputs shows how many times it was able to run this function (in the mean case) per second as a measure of performance and shows how this metric changes between commits.

![image](https://user-images.githubusercontent.com/23139455/101726008-8fe29b00-3a66-11eb-9724-ad782c187c41.png)

Reply:
@toumorokoshi That makes sense! What if we called it "history" though to confirm that this is with respects to commits (and not a load test or something that would have "time" on the x access)?

```suggestion
Performance history of benchmarks for packages distributed by OpenTelemetry Python can be viewed as a [graph of throughput vs commit history](https://open-telemetry.github.io/opentelemetry-python-contrib/benchmarks/index.html). From this page, you can download a JSON file with the performance results.
```